tests/test_writer.py

```diff
@@ -44,6 +44,29 @@
 SAMPLE_ROOT = Path(PROJECT_ROOT) / "sample-files"
 GHOSTSCRIPT_BINARY = shutil.which("gs")
 
+def test_fill_form_without_font(pdf_file_path):
```

Comment:
Please do not add new tests at the top. Additionally: Where can I see that the new functionality is checked? Shouldn't we test with your example file as well which would previously raise an exception?